fuzzers/klee/fuzzer.py

```diff
@@ -66,13 +67,23 @@ def emptydir(path):
     os.mkdir(path)
 
 
-def run(command, hide_output=False):
-    """Run a the command |command|"""
+def run(command, hide_output=False, ulimit_cmd=None):
+    """Run the command |command|, optionally, run |ulimit_cmd| first."""
     cmd = ' '.join(command)
     print('[run_cmd] {}'.format(cmd))
 
     output_stream = subprocess.DEVNULL if hide_output else None
-    ret = subprocess.call(command, stdout=output_stream, stderr=output_stream)
+    if ulimit_cmd:
+        ulimit_command = [ulimit_cmd + ';']
+        ulimit_command.extend(command)
+        ret = subprocess.call(' '.join(ulimit_command),
```

Comment:
Don't think the join call is needed. `call` should work on a list.

Reply:
I double checked, in general, call works with lists as you said - but I'm getting an issue where the side effects of ulimit -s aren't visible to klee (increased stack size) unless I do this join.